### PR TITLE
Changes for Callback url converts to string

### PR DESCRIPTION
**Pull request by kedarkhaire**
Fixes #apigee-edge-576 - https://github.com/apigee/apigee-edge-drupal/issues/576

### CODE REVIEW

File: apigee_devportal_kickstart.install [hunk 1]
```diff
@@ -704,3 +705,36 @@ function apigee_devportal_kickstart_update_9002() {
   // Clear all caches.
   drupal_flush_all_caches();
 }
+
+/**
+ * Updating install and optional configs for callback url.
+ */
+function apigee_devportal_kickstart_update_9003() {
```
Comment (divya-intelli) [inline apigee_devportal_kickstart.install]: As there is already an update script in [apigee-edge PR](https://github.com/apigee/apigee-edge-drupal/pull/913), do we have to add it again?

Comment (kedarkhaire) [reply apigee_devportal_kickstart.install]: As dicussed in call, I have removed the update [aa0744f](https://github.com/apigee/apigee-devportal-kickstart-drupal/pull/647/commits/aa0744ffa7035c4ae67d29fd016d6df2ff790948)

Comment (kedarkhaire) [reply apigee_devportal_kickstart.install]: done in update

**Review by divya-intelli — APPROVED** [review]
LGTM

**Pull request merged**